README.rst

```diff
@@ -3,14 +3,16 @@ The Ubuntu Packaging Guide
 
 This guide is often one of the first things that a new Ubuntu contributor will
 look at. Helping make it better can have a big impact.
-
 The actual articles can be found under the ``docs/`` directory in rst files.
 Images can be placed in the ``docs/images/`` folder.
+The html, js and css files can be found under ``docs/_static``. 
+
+You can find the full documentation [here](https://canonical-ubuntu-packaging-guide.readthedocs-hosted.com/).
```

Comment:
note: This already got addressed by PR #57

Reply:
note: The README is written in the [reStructuredText format](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html) and not in markdown, like most other repositories on GitHub.

This would be the correct link format in reStructuredText:
```suggestion
You can find the full documentation `here <https://canonical-ubuntu-packaging-guide.readthedocs-hosted.com/>`_.
```